lmms_eval/models/llava_vid.py

```diff
@@ -96,7 +108,6 @@ def __init__(
         self.mm_spatial_pool_mode = mm_spatial_pool_mode
         self.max_frames_num = int(max_frames_num)
         self.mm_resampler_location = mm_pooling_position
-        self.mm_newline_position = mm_newline_position
```

Comment:
The mm_newline_position error was introduced back again during merging for llava vid